File: sandbox/node/src/SignatureRequestCancelExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.SignatureRequestApi();
apiCaller.username = "YOUR_API_KEY";
// apiCaller.accessToken = "YOUR_ACCESS_TOKEN";

apiCaller.signatureRequestCancel(
  "fa5c8a0b0f492d768749333ad6fcc214c111e967", // signatureRequestId
).catch(error => {
  console.log("Exception when calling SignatureRequestApi#signatureRequestCancel:");
  console.log(error.body);
});

File: sandbox/ruby/src/SignatureRequestCancelExample.rb
require "json"
require "dropbox-sign"

Dropbox::Sign.configure do |config|
    config.username = "YOUR_API_KEY"
    # config.access_token = "YOUR_ACCESS_TOKEN"
end

begin
    Dropbox::Sign::SignatureRequestApi.new.signature_request_cancel(
        "fa5c8a0b0f492d768749333ad6fcc214c111e967", # signature_request_id
    )
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling SignatureRequestApi#signature_request_cancel: #{e}"
end
